### PR TITLE
Chapter5it

### PR DESCRIPTION
**Pull request by Nolanogenn**
Hi, I am working on the 5th chapter for the Italian translation of the course. I am still pretty new to Github, so if I am doing anything wrong please let me know!
Also if anyone wants to have a look at my translation please do.

### CODE REVIEW

**Review by lewtun — COMMENTED** [review]
Wow this is incredible work @Nolanogenn - awesome job translating a whole chapter and welcome to the 🤗 community!

There's just a small merge conflict that needs to be resolved in the `_toctree.yml` file and can you also remove the `.7.mdx.swp` file? Other that that, this should be good to go!

**Comment by Nolanogenn** [comment]
@lewtun I think I have fixed the issues, please tell me whether I am wrong!

**Comment by lvwerra** [comment]
Hi @Nolanogenn, there still seems to be a merge conflict in `_toctree.yml`. It can probably be fixed my merging `main` into your branch, (or you can click on `Resolve conflicts` at the bottom of the PR and do it manually). Let me know if you need any help!

**Comment by Nolanogenn** [comment]
Hi @lvwerra! Unfortunately it looks like I am having some issues getting through the checks now. Do you know what could be causing this?

File: chapters/it/chapter5/8.mdx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+<Question
+	choices={[
+		{
+			text: "In locale, ad esempio dal tuo portatile.",
+			explain: "Corretto! Puoi passare i path dei file nell'argomento <code>data_files</code> di <code>load_dataset()</code> per caricare dataset locali.",
+			correct: true
+		},
+		{
+			text: "L'Hub Hugging Face.",
+			explain: "Corretto! Puoi caricare i dataset presenti sull'Hub fornendo l'ID del dataset, ad esempio <code>load_dataset('emotion')</code>."
```
Comment (lvwerra) [inline chapters/it/chapter5/8.mdx:20]: Looking at the logs I think this is the culprit:

```suggestion
			explain: "Corretto! Puoi caricare i dataset presenti sull'Hub fornendo l'ID del dataset, ad esempio <code>load_dataset('emotion')</code>.",
```

Comment (Nolanogenn) [reply chapters/it/chapter5/8.mdx]: Thanks a lot for the help!

File: chapters/it/chapter5/2.mdx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+```python
+!gzip -dkv SQuAD_it-*.json.gz
+```
+
+```bash
+SQuAD_it-test.json.gz:	   87.4% -- replaced with SQuAD_it-test.json
+SQuAD_it-train.json.gz:	   82.2% -- replaced with SQuAD_it-train.json
+```
+
+Vediamo che i dati compressi sono stati sostituiti da _SQuAD_it-train.sjon_ e _SQuAD_it-text.sjon_, e che i dati sono archiviati in formato JSON.
```
Comment (lvwerra) [inline chapters/it/chapter5/2.mdx:49]:
```suggestion
Vediamo che i dati compressi sono stati sostituiti da _SQuAD_it-train.json_ e _SQuAD_it-text.json_, e che i dati sono archiviati in formato JSON.
```

**Review by lvwerra — APPROVED** [review]
Looks good! Just a minor typo to fix :)

**Review by lewtun — APPROVED** [review]
Thanks for the review @lvwerra and thank you @Nolanogenn for fixing the merge conflicts! This looks great to me, so merging it now :)

**Pull request merged**